src/nft-card-front.ts

```diff
@@ -316,58 +313,9 @@ export class NftCardFrontTemplate extends LitElement {
   private getButtonTemplate() {
     return html`
       <button @click="${(e: any) => this.eventHandler(e, 'view')}">
-        ${BTN_TEXT[ButtonType.Buy]}
+        buy this item ❯
```

Comment:
don't we normally uppercase the first letter?

Reply:
this element has text-transform: uppercase, so it's irrelevant here